Blog-projects/Widget-window-in-Qt-Quick-app/FontControlsQmlForm.qml

```diff
@@ -0,0 +1,51 @@
+import QtQuick
+import QtQuick.Controls
+import QtQuick.Layouts
+import WidgetWindowsInQtQuickApp as Cpp
+
+Window {
+    id: window
+    readonly property alias text: textField.text
+    readonly property string fontFamily: fontSelector.model[fontSelector.currentIndex]
+    signal toggleWidgetsWindow
+    height: 110
+    width: 300
+    minimumHeight: 110
+    minimumWidth: 260
+    visible: true
+    title: qsTr("Font Controls - Qt Quick")
+    color: systemPalette.window
+    SystemPalette {
+        id: systemPalette
+    }
+    ColumnLayout {
+        anchors.fill: parent
+        anchors.margins: 10
+        TextField {
+            id: textField
+            text: "KDAB"
+            focus: true
+            Layout.fillWidth: true
+        }
+        Button {
+            text: "Switch to Widgets Form"
```

Comment:
I noticed there is a mix of qStr usage, is that intentional?

Reply:
Good catch. Nope, I'll fix it.